web_hr_recruitment_label/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017-2023 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Website HR recruitment labels",
```

Comment:
Can you rename the directory from web_hr_recruitment_label to website_hr_reqruitment_label

Reply:
Done 👍